app/build.gradle.kts

```diff
@@ -0,0 +1,78 @@
+plugins {
+	id("com.android.application")
+	id("kotlin-android")
+	id("kotlin-android-extensions")
+}
+
+android {
+	compileSdkVersion(29)
+
+	defaultConfig {
+		// Android version targets
+		minSdkVersion(21)
+		targetSdkVersion(29)
+
+		// Release version
+		versionCode = 905
+		versionName = "0.11.0"
```

Comment:
Updating the version in a separate commit after this PR?

Reply:
Yes. It will be this PR, apiclient bump, and then version bump.

Reply:
Awesome!